src/psyclone/generator.py

```diff
@@ -581,23 +592,24 @@ def check_psyir(psyir, filename):
             f"found '{type(psyir.children[0]).__name__}'.")
 
 
-def add_builtins_use(fp2_tree):
-    '''Modify the fparser2 tree adding a 'use builtins' so that builtin kernel
+def add_builtins_use(fp2_tree, name):
+    '''Modify the fparser2 tree adding a 'use <name>' so that builtin kernel
     functors do not appear to be undeclared.
 
     :param fp2_tree: the fparser2 tree to modify.
     :type fp2_tree: py:class:`fparser.two.Program`
+    :param str name: the name of the use statement.
```

Comment:
It's picky but "...name of the module imported by the user statement."

Reply:
Modified.